config/cable.yml

```diff
@@ -6,5 +6,5 @@ test:
 
 production:
   adapter: redis
-  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
+  url: <%= ENV.fetch("REDIS_WORKER_URL") { "redis://localhost:6379/1" } %>
   channel_prefix: manage_courses_backend_production
```

Comment:
@darokel  Do we still use this ancient name?

Reply:
`manage_courses_backend_production`? only for the database